<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationWindow</class>
 <widget class="QDialog" name="CalibrationWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>603</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>560</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QSlider" name="slider_zero">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>500</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="check_raw_vals">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>560</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Calib matrix</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>1011</width>
     <height>261</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string>0</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>1</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>2</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>3</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>4</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>5</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Offset</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Curr Measure</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Max Measure</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Min Measure</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Diff Measure</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Newton</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Calib Bias</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Curr Bias</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="matrix">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>280</y>
     <width>242</width>
     <height>242</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>true</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>40</number>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>40</number>
   </attribute>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>New Row</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>530</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Matrix gain:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="editMatrixGain">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>530</y>
     <width>113</width>
     <height>23</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetCalibration">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>560</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Calibration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAutoOffsetAdj">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Automatic Offset Adj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveToEproom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to eeprom</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoadCalib">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load calibration file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveCalib">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Save calibration file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnImportCalibMatrix">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Import calibration matrix</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLoad">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>440</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetCalibBias">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>290</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Calibration Bias</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetCalibBias">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>320</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Calibration Bias</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetCurBias">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>350</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Set Current Bias</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResetCurBias">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>380</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Current Bias</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CalibrationWindow</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CalibrationWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
